docs/development/manual_tests.md

```diff
@@ -525,6 +526,241 @@ nextflow run main.nf -profile docker,test --outdir ./results/AR_dedup_merged -du
 nextflow run main.nf -profile docker,test --input ~/eager_dsl2_testing/input/only_PE/pe_only.tsv --outdir ./results/AR_dedup_merged_PE_only -dump-channels -ansi-log false --preprocessing_tool 'adapterremoval' --deduplication_tool 'dedup' --preprocessing_excludeunmerged -resume
 ```
 
+### METAGENOMICS
+
+#### Complexityfilter
+
+##### Test bbduk
+
+```bash
+#### Use bbduk to remove low complexity reads _without_ saving the intermediate files
+## Expect: NO additional directory created, but the files in the profiling directory contain the 'complexity' suffix
+nextflow run main.nf -profile test,docker \
+  --outdir ./out \
+  --run_metagenomics \
+  --metagenomics_profiling_tool krakenuniq \
+  --metagenomics_profiling_database CUSTOM_KRAKEN_DB \
+  --run_metagenomics_complexityfiltering \
+  --metagenomics_complexity_tool bbduk
+```
+
+```bash
+#### Use bbduk to remove low complexity reads _with_ saving intermediate files
+## Expect: Additional directory created 'metagenomics_screening/complexity_filter/bbduk' that contains the fastq files
+## with 'complexity' postfix and a bbduk.log file for each library
+nextflow run main.nf -profile test,docker \
+  --outdir ./out \
+  --run_metagenomics \
+  --metagenomics_profiling_tool krakenuniq \
+  --metagenomics_profiling_database CUSTOM_KRAKEN_DB \
+  --run_metagenomics_complexityfiltering \
+  --metagenomics_complexity_tool bbduk \
+  --metagenomics_complexity_savefastq
+```
+
+## Test prinseq
+
+```bash
+#### Use prinseq to remove low complexity reads _without_ saving the intermediate files
+## Expect: NO additional directory created, but the files in the profiling directory contain the 'complexity_good_out' postfix
+
+nextflow run main.nf -profile test,docker \
+  --outdir out \
+  --run_metagenomics \
+  --metagenomics_profiling_tool krakenuniq \
+  --metagenomics_profiling_database CUSTOM_KRAKEN_DB \
+  --run_metagenomics_complexityfiltering \
+  --metagenomics_complexity_tool prinseq
+```
+
+```bash
+#### Use prinseq to remove low complexity reads _with_ saving the intermediate files
+## Expect: Additional directory created 'metagenomics_screening/complexity_filter/prinseq' that contains the fastq files
+## with 'complexity_good_out' postfix and a 'complexity.log' file for each library
+
+nextflow run main.nf -profile test,docker \
+  --outdir out \
+  --run_metagenomics \
+  --metagenomics_profiling_tool krakenuniq \
+  --metagenomics_profiling_database CUSTOM_KRAKEN_DB \
+  --run_metagenomics_complexityfiltering \
+  --metagenomics_complexity_tool prinseq
+  --metagenomics_complexity_savefastq
+```
+
+#### Profiling
+
+##### metaphlan
+
+```bash
+## metaphlan with default parameters
+## Expect:
```

Comment:
Was this tested? I assume expected the output metaphlan mp3 profiles?

Reply:
yes, tested :)